src/ui/components/Rating/index.js

```diff
@@ -71,6 +71,7 @@ export class RatingBase extends React.Component {
       }
 
       return (
+        // eslint-disable-next-line react/jsx-key
```

Comment:
Ugh. I was afraid of this. There is a `key` defined in the `props` object but it looks like the linter isn't able to recognize it. Could you file an upstream bug?